src/sentry/conf/server.py

```diff
@@ -878,6 +878,8 @@ def create_partitioned_queues(name):
     "organizations:org-subdomains": False,
     # Enable access to more advanced (alpha) datascrubbing settings.
     "organizations:datascrubbers-v2": True,
+    # Enable the new teams page (beta)
+    "organizations:teams-page-v2": True,
```

Comment:
Defaulting to on is risky,  as it will automatically enable in production, on premise and single tenant scenarios.

Reply:
Thanks for the tip Mark ;-)